### ISSUE TITLE
Formatting issue when you run clitest on more than one test file, when you change IFS on a test

### ISSUE THREAD

**Issue by niloct** [aureliojargas/clitest#47]
Hi Aurelio,

I am using iTerm2 on Mac OS Catalina, and results when you run clitest on more than one test file don't show aligned.

As we found the issue, I'm sending the pull request.

Thanks.

**Comment by niloct** [comment]
Well, more debugging done!

The issue really has nothing to do with the OS.

When you modify `IFS` in a test, as mentioned on `clitest` README, it lasts for subsequent tests.

```
(...)
#36	IFS=+ && X="a+b+c+d+e" && set - $X && for letra; do echo $letra; done

     ok  fail  skip
  12 0 0                test_01_grep.sh
  1 0 0                test_02_od.sh
  3 0 0                test_03_seq.sh
  2 0 0                test_04_head.sh
  5 0 0                test_05_sort.sh
  1 0 0                test_06_tee.sh
  11 0 0                test_07_ch2.sh
  1 0 0                test_08_ch12.sh

OK: 36 of 36 tests passed
```

So the fix as you deduced is to rewrite IFS on each test run. Then:

```
(...)
#36	IFS=+ && X="a+b+c+d+e" && set - $X && for letra; do echo $letra; done

     ok  fail  skip
     12     -     -    test_01_grep.sh
      1     -     -    test_02_od.sh
      3     -     -    test_03_seq.sh
      2     -     -    test_04_head.sh
      5     -     -    test_05_sort.sh
      1     -     -    test_06_tee.sh
     11     -     -    test_07_ch2.sh
      1     -     -    test_08_ch12.sh

OK: 36 of 36 tests passed
```

**Comment by aureliojargas** [comment]
Excellent @niloct, thanks for the detailed information! 

Since your fix is setting `IFS` later in the code, after all the tests has been already executed, that shouldn't affect any test run.

**Comment by aureliojargas** [comment]
Fixed by #48
